### PR TITLE
Minor edit on UG to add purpose

### PR DESCRIPTION
**Pull request by nowknowing**
If Intriduction to TutorBuddy bloats up, can consider moving TOC up.
For now looks fine since TOC is still within one-page view.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,10 +3,23 @@ layout: page
 title: User Guide
 ---
 # TutorBuddy
+Welcome to the user guide for TutorBuddy!<br/>
+Are you a tuition teacher looking for a reliable way to keep track of
+your 1-to-1 classes? Needing a system to remind you about upcoming classes?
+Or efficient way to calculate your class incomes? <br/>
```
Comment (enhao25) [inline docs/UserGuide.md]: Hmmm maybe using "class" here might not be suitable as a class is usually associated with a group 1-many tuition. Just putting as "Or efficient way to calculate your incomes?" might be better.

**Review by enhao25 — COMMENTED** [review]
Only have 1 comment, other than that, its good to go!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -2,16 +2,29 @@
 layout: page
 title: User Guide
 ---
+# TutorBuddy
+Welcome to the user guide for TutorBuddy!<br/>
+Are you a tuition teacher looking for a reliable way to keep track of
+your 1-to-1 lessons? Needing a system to remind you about upcoming classes?
+Or an efficient way to calculate your class incomes? <br/>
```
Comment (enhao25) [inline docs/UserGuide.md]: Hi shion! I think you have updated the class at some other places. I think updating it here under class income will be good as well. (E.g. Lesson incomes instead class incomes)

**Review by enhao25 — COMMENTED** [review]
Hi shion! LGTM, just 1 comment.

File: docs/UserGuide.md (hunk 2 above)
Comment (nowknowing) [reply docs/UserGuide.md]: omg yes sry thanks for pointing out

**Review by JonahhGohh — APPROVED** [review]
LGTM

**Pull request merged**